homework18/src/ba/bitcamp/homework18/task01/SQLConn.java

```diff
@@ -0,0 +1,38 @@
+package ba.bitcamp.homework18.task01;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class SQLConn {
+	
+	Connection conn = null;
+	
+	/*
+	 * Constructor for SQL Connection to our database.
+	 */
+	public SQLConn() {
+		try {
+			Class.forName("org.sqlite.JDBC");
+			conn = DriverManager.getConnection("jdbc:sqlite:edindb.db");
```

Comment:
Okay sada vidim da si napravio neki util tip klase. Ali nisi bas zavrsio pricu posto je ovo trebalo da bude static tip metode. Pa da se moze reci SQLConn.getConnection().
Takodjer imas fulu koju je imao i Boris, a to je da ne zatvarate konekcije i statemente. Takodjer procitaj malo o PreparedStatement (on je ono sto ti treba za rada sa bazom ;))